sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -104,6 +104,14 @@ def foo_pipeline(tag: str, pull_image_policy: str):
     self.artifact_location = artifact_location
     return self
 
+  def add_op_transformer(self, transformer):
+    """Configures the op_transformers which will be applied to all ops in the pipeline.
+
+    Args:
+      transformer: a function that takes a ContainOp as input and returns a ContainerOp
```

Comment:
Interesting point. Currently we use the PipelineConf for pipeline level configurations. Since users are not creating multiple pipelines in the same file, we used the PipelineConf as a static default configuration for the current pipeline. It make sense to surface the pipeline op if users are using multiple pipelines in the file.

Reply:
I think I don't get what you mean.

A pipeline has some steps. These steps are Argo templates.
Currently the DSL supports 2 of these: the container and the resource templates. Container templates are represented by `ContainerOp` entity, while resource templates by `ResourceOp`.

Reply:
Maybe the way I expressed it created conflicts. I meant "an operator of the pipeline", an Op. That would be more precise.